tests/unittests/sources/test_hetzner.py

```diff
@@ -114,29 +113,28 @@ def test_read_data(
             ],
         )
 
-        self.assertTrue(m_readmd.called)
+        assert m_readmd.called
```

Comment:
To make this approach our policy, I think it is worth updating doc/rtd/development/testing.rst 

Let's replace 
  * ``m.assert_called()`` => ``assert 0 != m.call_count``

with 
  * ``m.assert_called()`` => ``assert m.called``

Reply:
Done.